at_client/lib/src/manager/sync_manager.dart

```diff
@@ -48,18 +50,18 @@ class SyncManager {
     if (preference.syncStrategy == SyncStrategy.scheduled && !_isScheduled) {
       _scheduleSyncTask();
     }
+    syncUtil = SyncUtil();
```

Comment:
Why not remove this line and change the declaration line above from `SyncUtil? syncUtil;` to `SyncUtil syncUtil = SyncUtil();` ?

Reply:
Then you could change all usages of `syncUtil?.` to just `syncUtil.`

Reply:
This changes in implemented and pushed the changes.